### PR TITLE
feat(Emberspark Games): add presence

### PR DESCRIPTION
**Pull request by ItsMateo20**
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
image
image
image

### CODE REVIEW

File: websites/E/Emberspark Games/presence.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
+const presence = new Presence({ clientId: "1196469492310618142" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Icon = "https://i.imgur.com/GVRJihz.png",
+	DesolationIcon = "https://i.imgur.com/TUZ9rwv.png",
+	ConsumerCatalog = "https://i.imgur.com/q3m7DhW.png",
```
Comment (theusaf) [inline websites/E/Emberspark Games/presence.ts:7]: Assets should be 512x512px. If needed, use online tools like waifu2x to upscale images.

File: websites/E/Emberspark Games/presence.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+						presenceData.smallImageKey = Assets.Question;
+						presenceData.smallImageText = "Reading a question...";
+						presenceData.state = `Reading: "${
+							activeQuestion.querySelector("h1").textContent
+						}"`;
+					}
+
+					break;
+				}
+				// No default
```
Comment (theusaf) [inline websites/E/Emberspark Games/presence.ts:136]:
```suggestion
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/E/Emberspark Games/presence.ts (hunk 1 above)
Comment (ItsMateo20) [reply websites/E/Emberspark Games/presence.ts:7]: they have to be exacly 512x512 or can be bigger?

Comment (theusaf) [reply websites/E/Emberspark Games/presence.ts:7]: They can be bigger, although according to current guidelines, they should technically be exactly 512x512px. When merging, they will be resized down to 512x512.

File: websites/E/Emberspark Games/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		"id": "630812692659044352",
+		"name": "itsmateo20"
+	},
+	"service": "Emberspark Games",
+	"description": {
+		"en": "Emberspark Games is a team of passionate gamers with a knack for creating engaging and immersive experiences. With their deep understanding of game design and storytelling, they are committed to delivering top-notch content that resonates with players worldwide."
+	},
+	"url": "emberspark.games",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/GVRJihz.png",
```
Comment (DarkVillager) [inline websites/E/Emberspark Games/metadata.json:13]:
```suggestion
	"logo": "https://i.imgur.com/w5E2gIj.png",
```

File: websites/E/Emberspark Games/presence.ts [hunk 4]
```diff
@@ -0,0 +1,159 @@
+const presence = new Presence({ clientId: "1196469492310618142" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Icon = "https://i.imgur.com/GVRJihz.png",
```
Comment (DarkVillager) [inline websites/E/Emberspark Games/presence.ts:5]:
```suggestion
	Icon = "https://i.imgur.com/w5E2gIj.png",
```

File: websites/E/Emberspark Games/presence.ts (hunk 1 above)
Comment (DarkVillager) [inline websites/E/Emberspark Games/presence.ts:7]:
```suggestion
	ConsumerCatalog = "https://i.imgur.com/FxjlP2Z.png",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by ItsMateo20** [comment]
Finished also added some buttons and also thank you @DarkVillager for changing some of the icons for me

**Comment by DarkVillager** [comment]
> Finished also added some buttons and also thank you @DarkVillager for changing some of the icons for me

I tried the last one, using an upscaler, nope, using an svg trace tool, nope. So yeah big oof

**Comment by ItsMateo20** [comment]
used waifu2x as @theusaf said for the desolation logo

> > Finished also added some buttons and also thank you @DarkVillager for changing some of the icons for me
> 
> I tried the last one, using an upscaler, nope, using an svg trace tool, nope. So yeah big oof

File: websites/E/Emberspark Games/presence.ts [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+	if (!currentPath[1]) {
+		presenceData.details = "Emberspark Games";
+		presenceData.largeImageKey = Assets.Icon;
+		presenceData.state = "Viewing the home page";
+		presenceData.buttons = [
+			{
+				label: "Check out our games/mods!",
+				url: "https://embersparkgames.com",
+			},
+		];
```
Comment (theusaf) [inline websites/E/Emberspark Games/presence.ts:27]: [Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that lead to main pages are not allowed.

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**